common/src/main/resources/assets/wynntils/lang/zh_tw.json

```diff
@@ -1,982 +1,1001 @@
-  "feature.wynntils.itemStatInfo.defective.name": "混淆的故障物品名",
-  "feature.wynntils.itemStatInfo.groupIdentifications.description": "要是否把不同的屬性基於類型分組嗎？",
-  "feature.wynntils.itemStatInfo.groupIdentifications.name": "分組屬性",
-  "feature.wynntils.itemStatInfo.name": "物品數據",
-  "feature.wynntils.itemStatInfo.obfuscationChanceEnd.description": "故障物品每字最低機會被混淆是什麼？",
-  "feature.wynntils.itemStatInfo.obfuscationChanceEnd.name": "混淆字最高機會",
-  "feature.wynntils.itemStatInfo.obfuscationChanceStart.description": "故障物品每字最高機會被混淆是什麼？",
-  "feature.wynntils.itemStatInfo.obfuscationChanceStart.name": "混淆字最高機會",
-  "feature.wynntils.itemStatInfo.perfect.description": "要是否完美物品 (100%) 的名使用彩色顯示？",
-  "feature.wynntils.itemStatInfo.perfect.name": "彩色的完美物品名",
-  "feature.wynntils.itemStatInfo.reorderIdentifications.description": "要是否把物品的屬性基於 Wynntils 的次第安排嗎？",
-  "feature.wynntils.itemStatInfo.reorderIdentifications.name": "從安排屬性",
-  "feature.wynntils.itemStatInfo.showStars.description": "屬性字以後要是否有星星 (*，**，***) 嗎？",
+  "feature.wynntils.itemGuess.showGuessesPrice.description": "在未鑑定物品的提示中顯示價格",
+  "feature.wynntils.itemGuess.showGuessesPrice.name": "顯示推測價格",
+  "feature.wynntils.itemHighlight.cosmeticHighlightEnabled.description": "啟用時裝物品的醒目提示",
+  "feature.wynntils.itemHighlight.cosmeticHighlightEnabled.name": "時裝物品醒目提示",
+  "feature.wynntils.itemHighlight.craftedHighlightColor.description": "製作的物品應該使用什麼顏色醒目提示?",
+  "feature.wynntils.itemHighlight.craftedHighlightColor.name": "製作物品醒目提示顏色",
+  "feature.wynntils.itemHighlight.craftedHighlightEnabled.description": "啟用製作物品的醒目提示",
+  "feature.wynntils.itemHighlight.craftedHighlightEnabled.name": "製作物品醒目提示",
+  "feature.wynntils.itemHighlight.emeraldPouchHighlightEnabled.description": "啟用綠寶石布袋的醒目提示",
+  "feature.wynntils.itemHighlight.emeraldPouchHighlightEnabled.name": "綠寶石布袋醒目提示",
+  "feature.wynntils.itemHighlight.fabledHighlightColor.description": "Fabled 物品應該要用什麼顏色的醒目提示?",
+  "feature.wynntils.itemHighlight.fabledHighlightColor.name": "傳說物品醒目提示顏色",
+  "feature.wynntils.itemHighlight.fabledHighlightEnabled.description": "啟用 Fabled 物品的醒目提示",
+  "feature.wynntils.itemHighlight.fabledHighlightEnabled.name": "傳說物品醒目提示",
+  "feature.wynntils.itemHighlight.hotbarHighlightEnabled.description": "啟用快捷欄中物品的醒目提示",
+  "feature.wynntils.itemHighlight.hotbarHighlightEnabled.name": "快捷欄物品醒目提示",
+  "feature.wynntils.itemHighlight.hotbarOpacity.description": "快捷欄中醒目提示的物品應該使用什麼層級的不透明度?",
+  "feature.wynntils.itemHighlight.hotbarOpacity.name": "快捷欄醒目提示透明度",
+  "feature.wynntils.itemHighlight.inventoryHighlightEnabled.description": "啟用物品欄中物品的醒目提示",
+  "feature.wynntils.itemHighlight.inventoryHighlightEnabled.name": "物品欄物品醒目提示",
+  "feature.wynntils.itemHighlight.inventoryOpacity.description": "物品欄中醒目提示的物品應該使用什麼層級的不透明度?",
+  "feature.wynntils.itemHighlight.inventoryOpacity.name": "物品欄醒目提示不透明度",
+  "feature.wynntils.itemHighlight.legendaryHighlightColor.description": "Legendary 物品應該要用什麼顏色的醒目提示?",
```

Comment:
> The strings that have proper noun(ex: Item Rarities, Perfect prifix or Item Corkian Amplifier) I only translate the `xxx.name`, and keep `xxx.description` in English to prevent user being confuse, if need translates please let me know.

Yes, I want to use Chinese in `Name` and English in `Description`, if user don't know what tier mean in Chinese then they can check in description.